src/index.html

```diff
@@ -7,8 +7,74 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@200;400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a 
+        href="#home"
+        class="header__logo"
+      >
+        <img 
+          class="logo" 
+          src="./images/logo.png" 
+          alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a 
+              href="#apple"
+              class="nav__link is-active"
+            >Apple</a>
+          </li>
+          <li class="nav__item">
+            <a 
+            href="#Samsung"
+            class="nav__link"
+          >Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="#Smartphones"
+              class="nav__link"
+            >Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a 
+              class="nav__link"
+              data-qa="hover"
+              href="#Laptops & Computers"
+            >Laptops & Computers</a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="#Gadgets"
+              class="nav__link"
+            >Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="#Tablets"
+              class="nav__link"
+            >Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="#Photo"
+              class="nav__link"
+            >Photo</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a 
              href="#Tablets"
              class="nav__link"
            >Tablets</a>
          </li>

          <li class="nav__item">
            <a 
              href="#Photo"
              class="nav__link"
            >Photo</a>
          </li>
```

Add blank line between multiple sibling blocks